kubetest2-gke/deployer/deployer.go

```diff
@@ -79,6 +79,12 @@ type ig struct {
 	uniq string
 }
 
+type cluster struct {
+	// index is the index of the cluster in the list provided via the --cluster-name flag
+	index int
+	name  string
+}
```

Comment:
this is a good start, we should continue to move other attributes inside this struct, to construct an object with all relevant creation info.